src/renderers/webgl/WebGLLights.js

```diff
@@ -407,8 +407,34 @@ function WebGLLights() {
 
 		if ( rectAreaLength > 0 ) {
 
-			state.rectAreaLTC1 = UniformsLib.LTC_1;
-			state.rectAreaLTC2 = UniformsLib.LTC_2;
+			if ( capabilities.isWebGL2 ) {
```

Comment:
@mrdoob This checks will be done per frame although the capabilities and extensions won't change over time^^.

Given the experimental status of `RectAreLight`, I think it's okay to do it this way. If we want to avoid these checks, we would to enhance this code so it picks the correct data textures only once.